### ISSUE TITLE
No way to specify search path for use/include [$5]

### ISSUE THREAD

**Issue by ex-nerd** [openscad/openscad#604]
I've started working with a set of shared libraries, and with no way to specify a search path for them (just a single hard-coded directory), I'm relegated to doing something like this for every single include file:

```
// Not sure where the main dotscad files might be, so try to load from a few locations.
use <../../dotscad/pie.scad>;
use <../dotscad/pie.scad>;
use <dotscad/pie.scad>;
use <pie.scad>;
```

It would be really nice if there was a way to specify a PATH-style search path for includes (preferably in the scad source itself, if not via an environment variable like $PATH).  Then scad source could just look something like:

```
set_include_path "./:../:../../";
use <dotscad/pie.scad>;
use <otherlib.scad>;
```
##

**Comment by t-paul** [comment]
I don't think that will be implemented. @donbright would you re-claim you bounty when we close this?
